jwst/regtest/test_nirspec_fs_spec3.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+@pytest.fixture(scope="module")
+def run_pipeline(jail, rtdata_module):
+    """
+    Run the calwebb_spec3 pipeline on NIRSpec Fixed-Slit exposures.
+    """
+
+    rtdata = rtdata_module
+
+    # Get the cfg files
+    collect_pipeline_cfgs("config")
+
+    # Get the ASN file and input exposures
+    rtdata.get_asn('nirspec/fs/jw93045-o010_20180725t035735_spec3_001_asn.json')
+
+    # Run the calwebb_spec3 pipeline; save results from intermediate steps
+    args = ["config/calwebb_spec3.cfg", rtdata.input,
+            "--steps.outlier_detection.save_results=true",
+            "--steps.resample_spec.save_results=true",
```

Comment:
This `resample_spec.save_results=True` should not be necessary, right?  The pipeline itself makes sure this is written out.  Though maybe no harm if it is there?

Reply:
That was my thought too: it's already hardwired in the ``calwebb_spec3.cfg`` file and potentially in the pipeline/step code, but I figured it can't hurt to make doubly sure it's set.